server/rosetta/launchpad.go

```diff
@@ -0,0 +1,82 @@
+package rosetta
```

Comment:
Couldn't this package become a top-level one? I mean: `git mv server/rosetta rosetta/`